components/gallery/GalleryItem.vue

```diff
@@ -98,6 +98,12 @@ const CarouselTypeVisited = defineAsyncComponent(
 </script>
 
 <style lang="scss" scoped>
+@import '@/styles/abstracts/variables';
+
+.title {
+  font-size: 39px;
```

Comment:
all good here? 👀

Reply:
yeah sort of `2.4375em` seem a bit random to me, you could have use `is-2` which is equivalent to `2.5`

Reply:
@exezbcz was quite specific 😆 
![image](https://user-images.githubusercontent.com/73316633/210756209-c7a66fd5-18b8-4829-a4b6-0bb49962ff95.png)

Reply:
`px` is really not the best option for responsive :)

Reply:
I meant it's `2.4375em`, which is `39px` on the desktop; let me know if I should change it (I'm okay with `40px`  or `is-2` / `2.5em`). This is the size that was in the issue description, but I don't have a preference here.

Reply:
Sure no worries. It just confuses me that we'll have a particular `title` size only for this page. On the other hand, if we are going to reuse "Heading 2" type size it would be better to directly change `$size-2` value